server/api/constants.go

```diff
@@ -37,8 +37,8 @@ const (
 	KVPath          = "/kv"
 	SubscribePath   = "/subscribe"
 	UnsubscribePath = "/unsubscribe"
-
-	BindingsPath = "/bindings"
+	AssetPath       = "/asset"
```

Comment:
1/5 cc @hanzei should this be `/static`, as in the plugins?

Reply:
Good question. I like `static` because it indicates the nature of the files. So there would be  `static assets` served under `/static` and `dynamic assets` generated and served by the app via the lambda.